src/prefect/engine.py

```diff
@@ -1026,12 +1045,19 @@ async def _in_process_pause(
 
     logger.info("Pausing flow, execution will continue when this flow run is resumed.")
 
+    proposed_state = Paused(
+        timeout_seconds=timeout, reschedule=reschedule, pause_key=pause_key
+    )
+
+    run_input_keyset = keyset_from_paused_state(proposed_state)
```

Comment:
Do we need this if we aren't waiting for input?

Reply:
Nope.